### ISSUE TITLE
Does it work with NativeScript+Vue?

### ISSUE THREAD

**Issue by bonustrack** [NathanaelA/nativescript-websockets#42]
I'm trying to enable WebSocket client on a NativeScript+Vue project. I got `require('nativescript-websockets');` on the main file of the project (main.js) but it doesn't seem to work, i can't get ride of this error: 
```
file:///app/app.js:1:197110: JS ERROR Error: Could not find module 'nativescript-websockets/websockets-common'. Computed path '/Users/fabien/Library/Developer/CoreSimulator/Devices/50013167-A53B-4682-9E95-77A95281E43E/data/Containers/Bundle/Application/D717CE39-6818-4866-A50C-65D27C7F589A/dist.app/app/tns_modules/nativescript-websockets/websockets-common'.
```

Any idea on how to make `nativescript-websockets` working on NativeScript+Vue?

**Comment by joshporter1** [comment]
Getting the same error. Were you able to get this working?

**Comment by bonustrack** [comment]
@joshporter1 yes but i don't remember how i did that. Did you reset your `node_module` and `dist` folders before adding `nativescript-websockets`? Do you using `vue-cli` with template 2.0?

**Comment by joshporter1** [comment]
@bonustrack I was using vue-cli. Turns out I missed the step to run `npm run clean` after installing a plugin. I assume this is what you did too when you made this issue? Did you have any other issues, or should this issue be marked as resolved?
